.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
```

Comment:
@jrburke, I also added `sudo: false`, as it's supposedly a bit quicker, and nicer to caching.

http://docs.travis-ci.com/user/workers/container-based-infrastructure/